cephfs/file.go

```diff
@@ -199,3 +201,32 @@ func (f *File) Fchown(user uint32, group uint32) error {
 	ret := C.ceph_fchown(f.mount.mount, f.fd, C.int(user), C.int(group))
 	return getError(ret)
 }
+
+// FallocFlags represent flags which determine the operation to be
+// performed on the given range.
+// CephFS supports only following two flags.
+type FallocFlags int
+
+const (
+	// FallocFlKeepSize specifies that the file size will not be changed.
+	FallocFlKeepSize = int(FallocFlags(unix.FALLOC_FL_KEEP_SIZE))
```

Comment:
You do not need to go-import `unix`. Just `#include <fcntl.h>` and use `C.FALLOC_FL_KEEP_SIZE`. This keeps the go-lang dependencies to a minimum.

Reply:
Also, I do not think it is needed to convert this to an `int` here.

Reply:
Ok, I was also confused whether I should expose these as FallocFlags or should I directly use these as C.FALLOC_FL_*. 
Please suggest, have exposed these because cephfs supports only these two flags.

Reply:
@nixpanic Ignore my previous comment, I think we should expose it.

But I don't think FALLOC_FL_* is there in fcntl.h, I am getting compilation error while using the flags via fcntl.h

Reply:
I have them included, `fcntl.h` includes (eventually) `/usr/include/linux/falloc.h` on my system (Fedora 31). What distribution are you using?

If the distribution you are running on is current enough, we may need to depend on importing the go-lang external unix package...

Reply:
Mine is also Fedora 31 but I checked that my fcntl.h doesn't include falloc.h :(

I guess we need to include like <linux/falloc.h> but not sure that whether it will work every where or not.

Reply:
You might need to add `#define _GNU_SOURCE` before the `#include <fcntl.h>` line

Reply:
Thanks @nixpanic, it works.